docs/guides/Guide-Nested.md

```diff
@@ -72,3 +72,43 @@ Now we have put one navigator inside another, and we can `navigate` between navi
 ```phone-example
 nested
 ```
+
+## Side Note - Nesting a Navigator in a Component
```

Comment:
Nit: I think you can omit the phrase "Side Note." This is might be confusing for people who are not fluent in English. The documentation also does not currently use the phrase anywhere else. If it's valuable information to a developer, just drop "side note" :)